setup.py

```diff
@@ -45,5 +51,13 @@
         "defconQt.representationFactories",
         "defconQt.util",
     ],
-    package_dir={"": "Lib"}
+    scripts=["trufont"],
```

Comment:
@n7s 

you need to add a `main()` function in `__main__.py` module, like this:

```python

def main():
    # indent the content of __main__.py module

if __name__ == "__main__":
    main()
```

and then you add the following `entry_points` to `setup`:

```python
entry_points={
        'console_scripts': [
            "trufont = defconQt.__main__:main"
            ],
        },
```